Library/Formula/multimarkdown.rb

```diff
@@ -8,13 +8,6 @@ class Multimarkdown < Formula
 
   head "https://github.com/fletcher/MultiMarkdown-5.git"
 
-  bottle do
```

Comment:
How can I then test the compilation and installation? Running ```brew install --verbose --debug --install-from-source multimarkdown``` still uses the bottle (```Downloading https://homebrew.bintray.com/bottles/multimarkdown-5.0.el_capitan.bottle.tar.gz```)...